source/settings/plugin/fts-solr-plugin.rst

```diff
@@ -12,69 +12,68 @@ Settings
-
-       * Remember to add your core name if using solr 7+:
-         ``"/solr/dovecot/"``.
+.. dovecot_plugin:setting:: fts_solr_batch_size
+   :plugin: fts-solr
+   :values: @uint
+   :default: 1000
+   :added: 2.4.0,3.0.0
 
-   ``debug``
+   Configures the number of mails sent to Solr in a single request.
 
-     * Default: no
-     * Values:  <existence> (if ``debug`` exists, it is enabled)
-     * Enable HTTP debugging. Writes to debug log.
+   * With ``fts_autoindex=yes``, each new mail gets separately indexed on
+     arrival, so ``batch_size`` only matters during the initial indexing
+     of a mailbox.
 
-   ``rawlog_dir=<directory>``
+   * With ``fts_autoindex=no``, new mails don't get indexed on arrival,
+     so ``batch_size`` is used when indexing is triggered.
 
-     .. dovecotadded:: 2.3.6
+.. dovecot_plugin:setting:: fts_solr_debug
+   :plugin: fts-solr
+   :values: @boolean
+   :default: no
+   :added: 2.4.0,3.0.0
 
-     * Default: <empty>
-     * Values:  :ref:`string`
-     * Enable rawlog debugging. Store HTTP exchanges between Dovecot and Solr
-       in this directory.
+   Enables HTTP debugging. Writes to debug log.
 
-   ``batch_size``
+.. dovecot_plugin:setting:: fts_solr_rawlog_dir
+   :plugin: fts-solr
+   :values: @string
+   :default: <empty>
+   :added: 2.4.0,3.0.0
 
-     .. dovecotadded:: 2.3.6
+   Enables rawlog debugging. Stores HTTP exchanges between Dovecot and Solr
+   in this directory.
 
-     * Default: ``1000``
-     * Values:  :ref:`uint`
-     * Configure the number of mails sent in single requests to Solr.
+.. dovecot_plugin:setting:: fts_solr_soft_commit
+   :plugin: fts-solr
+   :values: @boolean
+   :default: yes
+   :added: 2.4.0,3.0.0
 
-       * With ``fts_autoindex=yes``, each new mail gets separately indexed on
-         arrival, so ``batch_size`` only matters when doing the initial
-         indexing of a mailbox.
-       * With ``fts_autoindex=no``, new mails don't get indexed on arrival, so
-         ``batch_size`` is used when indexing gets triggered.
+   Controls whether new mails are immediately searchable via Solr.
 
-   ``soft_commit=yes|no``
+   * See :ref:`fts_backend_solr-soft_commits` for additional information.
 
-     .. dovecotadded:: 2.3.6
+.. dovecot_plugin:setting:: fts_solr_url
+   :plugin: fts-solr
+   :values: @string
+   :default: https://localhost:8983/solr/dovecot/
+   :added: 2.4.0,3.0.0
 
-     * Default: ``yes``
-     * Values:  :ref:`boolean`
-     * Controls whether new mails are immediately searchable via Solr.
+   Required base URL for Solr.
 
-       * See :ref:`fts_backend_solr-soft_commits` for additional information.
+   * Remember to add your core name if using solr 7+:
+     ``"/solr/dovecot/"``.
 
-   Example:
+Example:
 
-   .. code-block:: none
+.. code-block:: none
 
-     plugin {
-       fts_solr = url=http://solr.example.org:8983/solr/ debug batch_size=1000
-     }
+  fts_solr_uri = http://solr.example.org:8983/solr/dovecot/
+  fts_solr_debug = yes
```

Comment:
remove. Although could mention somewhere that `log_debug = category=fts-solr` can be used for debugging.